.github/workflows/pull-request.yml

```diff
@@ -18,8 +18,9 @@ jobs:
           node-version: 12
       - name: Install Dependencies
         run: yarn install
+      - name: Lint
+        run: yarn lint && npx commitlint --from HEAD~1 --to HEAD
```

Comment:
Will this not only validate one commit? because we are comparing the current HEAD minus 1 with the HEAD which should be master, no?

Reply:
oh... you are right, I will have to check all the commits starting from the last release 🤦‍♂️

Reply:
changed it to use `${{github.event.pull_request.base.sha}}`